tests/secrets.sample.yaml

```diff
@@ -23,6 +23,15 @@ placement_password: CUSTOMIZE_THIS
 
 # Controller SSH connection strings for the MariaDB copy procedure.
 # Use ":" for controler 2 and 3 if you are testing with a single controller.
-controller1_ssh: "ssh -F ~/director_stanalone/vagrant_ssh_config vagrant@standalone"  # CUSTOMIZE THIS
+controller1_ssh: "ssh -F ~/director_standalone/vagrant_ssh_config vagrant@standalone"  # CUSTOMIZE THIS
 controller2_ssh: ":"
 controller3_ssh: ":"
+
+# virsh --connect=qemu:///system -q domifaddr director_standalone | awk 'NF>1{print $NF}' | cut -d/ -f1
+edpm_node_ip: CUSTOMIZE_THIS
+
+# Ensure vagrant private key works for root
+# ssh -F ~/director_standalone/vagrant_ssh_config vagrant@standalone sudo cp .ssh/authorized_keys /root/.ssh/authorized_keys
+# path to the vagrant private key:
+# ~/director_standalone/.vagrant/machines/standalone/libvirt/private_key
+edpm_privatekey_path: CUSTOMIZE_THIS
```

Comment:
as the way of capturing the IP and private key will be different from CI, I stopped trying to capture them and moved the vars to here

Reply:
:+1: perfect i think this is the way to go.